src/models/User.ts

```diff
@@ -1,28 +1,32 @@
+import {
+  Grade, Role, ScholarshipType, Gender,
+} from './enums';
+
 interface UserProps {
-    age: number;
-    schoolYear: number;
-    role: string;
-    gender: string;
-    scholarshipPreferences: {
-        type: string,
-        major: string,
-        amount: number, 
-        school: string,
-        grade: number,
-        location: string
-    };
-    otherSettings: {
-        notifications: boolean,
-        deadlineNotifications: boolean,
-        relatedNotifications: boolean
-    }
+  age: number;
```

Comment:
This will get outdated within a year. Maybe instead store the birth date? Or if we don't want to ask for a specific date we can at least ask for the birth year and possibly the birth month too